scripts/deploy.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const chalk = require('chalk');
+const package = require("../package.json");
+const { exec } = require("child_process");
+
+if (package.config.mode.frontend != package.config.mode.backend) {
+    console.error(chalk.red(`Frontend is point to ${package.config.mode.frontend} server and backend is pointing to ${package.config.mode.backend} server. Make sure you know what your doing!`));
```

Comment:
Nit: "front end is pointing to" not point to :P

Reply:
LOL, correcting that. :P